compiler/nim.cfg

```diff
@@ -5,7 +5,6 @@ path:"llvm"
 path:"$projectPath/.."
 
 define:booting
-#import:"$projectpath/testability"
```

Comment:
I just removed it in a habit to remove commented code. But if it causes discussion I reanimate it.